frontend/packages/git-service/src/services/bitbucket-service.ts

```diff
@@ -150,6 +150,8 @@ export class BitbucketService extends BaseService {
   isDockerfilePresent = () =>
     this.isFilePresent(`${this.metadata.contextDir}/${this.metadata.dockerfilePath}`);
 
+  isTektonFolderPresent = () => this.isFilePresent(`${this.metadata.contextDir}/.tekton`);
```

Comment:
Good to add unit tests in git service for the presence of .tekton folder.

Reply:
Thanks. Found out that the current implementation does not work for Gitlab/Bitbucket PAC repos. Need to change some codes.